client/scss/components/_footer_nav.scss

```diff
@@ -1,5 +1,7 @@
 .footer-nav {
   color: color('white');
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Why only on the footer? Is it because of the dark background?
If so could we not have a `.dark-background` class that:
```
.dark-background {
  background-color: dark;
  webkit-font-smoothing: antialiased;
  moz-osx-font-smoothing: grayscale;
}
```